packages/syft/src/syft/core/adp/data_subject_list.py

```diff
@@ -584,6 +590,9 @@ def var(self, *args: List[Any], **kwargs: Dict[Any, Any]) -> DataSubjectArray:
     def sqrt(self, *args: List[Any], **kwargs: Dict[Any, Any]) -> DataSubjectArray:
         return DataSubjectArray(self.data_subjects)
 
+    def mean(self) -> DataSubjectArray:
+        return DataSubjectArray(self.data_subjects)
```

Comment:
@IshanMi It doesn't look like the `DataSubjectArray` class has a `sum` method yet. Should this be something more like the `var` method? https://github.com/OpenMined/PySyft/blob/106a5ba193f3fc39594d3ddba4c1fd8dfbb5a84c/packages/syft/src/syft/core/adp/data_subject_list.py#L581-L582

Reply:
MAYBE: We could remove mean method,The mean method would work automatically, like  since we covered the basic operations of mean  (addition and division).calling np.mean would invoke add and div automatically

Ex:

<pre>
<code>
from syft.core.adp.data_subject_list import DataSubjectArray

natsu = DataSubjectArray(["natsu"])
zerek = DataSubjectArray(["zerek"])

data_subjects = np.array([natsu,zerek])

res = data_subject.mean() or np.mean(data_subjects)
print(res)
</code>
</pre>